### DISCUSSION TITLE
群组的层级关系

### DISCUSSION THREAD

**Discussion by ChenSheng233** [antvis/X6#3303] [Q&A]
在使用 X6 图编辑引擎的时候，有这么一个场景。在一个 parent Node 节点里面有若干 child node ，将外部的 Node 移动到
Parent Node 里面时，会出现被遮挡的情况。请问要怎么让“背后”的Node 放到“顶层”。
![D265FE3F7C2E49E990286E1CC260210D 756](https://user-images.githubusercontent.com/39158941/221176336-cdecc05a-6cc1-434b-b44c-c9ea38a5fd7b.png)

**Comment by NewByVector** [comment]
可以设置  zIndex 属性。
